### DISCUSSION TITLE
Filter expressions like max,min,avg, etc?

### DISCUSSION THREAD

**Discussion by fgreco55** [milvus-io/milvus#27790] [Q&A and General discussion]
From the docs, I don't see any query expressions like max(), min(), avg(), et al.  
Is that something planned in a future release?

**Comment by yhmo** [comment]
Eventually, these built-in methods will definitely be implemented in Milvus. Currently, we are working on "group by".

**Comment by xiaofan-luan** [comment]
@fgreco55 
is there use case for min and max ? We'd love to know more details about the use case
